css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,42 @@
+.container {
+  width: 1080px;
+  margin: 0 auto;
+  font-family: 'DM Sans', sans-serif;
+}
+
+.row {
+  display: flex;
+  align-items: baseline;
+  height: auto;
+}
+
+.col-5 {
+  width: 20%;
+  margin: 0 10px 5px 10px;
+  font-size: small;
+}
+
+.col-full {
+  width: 100%;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
I'm not sure you actually need these `margin-` properties. Go through all of your CSS and make sure the properties actually have an effect.